app/lib/frontend/templates/report.dart

```diff
@@ -20,6 +20,7 @@ const _subjectKindLabels = {
 String renderReportPage({
   SessionData? sessionData,
   required ModerationSubject subject,
+  required String? url,
```

Comment:
This should also be added to `lcpsDeepLink` as `&url=` in querystring.

Reply:
I think we need `subject.canonicalUrl` for that: https://github.com/dart-lang/pub-dev/blob/master/app/lib/frontend/templates/report.dart#L31
(Also: I have an almost ready PR for that, submitting shortly)

Reply:
Ah, misremembered this, follow-up in: https://github.com/dart-lang/pub-dev/pull/7713